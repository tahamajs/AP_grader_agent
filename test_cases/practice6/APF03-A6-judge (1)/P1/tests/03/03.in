POST signup ? username "low_mist" password "meow"
POST logout ?
POST signup ? username "cool_man" password "1234"
POST logout ?
POST signup ? username "amir_akbar" password "1382"
GET districts ?
POST logout ?
POST login ? username "cool_man" password "1234"
GET districts ? district "SatarKhan"
GET districts ? district "Meydan Sanat"
POST logout ?
POST login ? username "low_mist" password "meow"
GET districts ?
GET districts ? district "Mirdamad"
POST logout ?