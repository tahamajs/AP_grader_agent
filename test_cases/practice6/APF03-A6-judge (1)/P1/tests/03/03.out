OK
OK
OK
OK
OK
Dowlat Abad: Payane Jonoub, Pirouzi
Enghelab: Karegar, Payane Jonoub, Pirouzi
Hossein Abad: Mirdamad, Tehranpars, Tajrish, Pirouzi
Karegar: Enghelab, Meydan Sanat, Mirdamad
Meydan Sanat: Karegar, Mirdamad, Tajrish
Mirdamad: Hossein Abad, Tajrish, Meydan Sanat, Karegar
Payane Jonoub: Dowlat Abad, Pirouzi, Enghelab
Pirouzi: Tehranpars, Payane Jonoub, Dowlat Abad, Hossein Abad, Enghelab
Tajrish: Mirdamad, Meydan Sanat, Hossein Abad
Tehranpars: Hossein Abad, Pirouzi
OK
OK
Not Found
Meydan Sanat: Karegar, Mirdamad, Tajrish
OK
OK
Dowlat Abad: Payane Jonoub, Pirouzi
Enghelab: Karegar, Payane Jonoub, Pirouzi
Hossein Abad: Mirdamad, Tehranpars, Tajrish, Pirouzi
Karegar: Enghelab, Meydan Sanat, Mirdamad
Meydan Sanat: Karegar, Mirdamad, Tajrish
Mirdamad: Hossein Abad, Tajrish, Meydan Sanat, Karegar
Payane Jonoub: Dowlat Abad, Pirouzi, Enghelab
Pirouzi: Tehranpars, Payane Jonoub, Dowlat Abad, Hossein Abad, Enghelab
Tajrish: Mirdamad, Meydan Sanat, Hossein Abad
Tehranpars: Hossein Abad, Pirouzi
Mirdamad: Hossein Abad, Tajrish, Meydan Sanat, Karegar
OK